mongodb/src/main/java/com/yahoo/ycsb/db/OptionsSupport.java

```diff
@@ -76,7 +76,7 @@ else if ("acknowledged".equals(writeConcernType)) {
                 result = addUrlOption(result, "w", "1");
             }
             else if ("journaled".equals(writeConcernType)) {
-                result = addUrlOption(result, "journal", "true");
+                result = addUrlOption(result, "j", "true");
```

Comment:
It never has, actually.  I reported it as https://jira.mongodb.org/browse/JAVA-1890, but for now let's switch to "j".